src/components/Petitions/index.js

```diff
@@ -1,38 +1,43 @@
 import React from 'react';
 import settings from 'settings';
+import { Link } from 'react-router';
 import Container from 'components/Container';
 import Grid from 'components/Grid';
 import GridItem from 'components/GridItem';
+import TextCenter from 'components/TextCenter';
 import Teaser from 'components/Teaser';
 import Header from 'components/Header';
 import PageTitle from 'components/PageTitle';
-import ButtonLink from 'components/ButtonLink';
+import Autocomplete from 'containers/Autocomplete';
 import Loading from 'components/Loading';
 
-const Petitions = ({ petitions }) => (
+const Petitions = ({ petitions, autocompleteProps }) => (
   <Container>
     <section>
       <Header>
         <PageTitle
           title={settings.petitionsPage.title}
           centered>
-          <ButtonLink
-            href={'/petitions/new'}
-            text={settings.petitionsPage.createButton.text}
-            modifier={'accent'}
-          />
+
+          <Autocomplete {...autocompleteProps} />
         </PageTitle>
       </Header>
+
       <Loading isLoading={petitions.isLoading}>
-        <Grid>
-          {petitions.map((petition) => {
-            return (
-              <GridItem key={petition.id}>
-                <Teaser {...petition} key={petition.id} />
-              </GridItem>
-            );
-          })}
-        </Grid>
+        {petitions.length
+          ? <Grid>
+            {petitions.map((petition) => {
+              return (
+                <GridItem key={petition.id}>
+                  <Teaser {...petition} key={petition.id} />
+                </GridItem>
+              );
+            })}
+          </Grid>
+          : <TextCenter>
+            There are no petitions for this city yet! <Link to='/petitions/new'>Create the first!</Link>
+          </TextCenter>
```

Comment:
I've done the fallback etc. logic in the pagination branch, but we will need to align.